src/azure-cli/azure/cli/command_modules/vm/manual/_help.py

```diff
@@ -57,3 +57,76 @@
     type: command
     short-summary: "Delete an SSH public key."
 """
+
+helps['sig share'] = """
+    type: group
+    short-summary: Manage gallery sharing profile
+"""
+
+helps['sig share add'] = """
+    type: command
+    short-summary: "Update sharing profile of a gallery."
```

Comment:
```suggestion
    short-summary: Add sharing profile of a gallery.
```

Reply:
It is updating share profile with adding new sharing id. I will refine it

Reply:
OK~